bigbluebutton-html5/imports/ui/components/video-dock/video-menu/container.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { createContainer } from 'meteor/react-meteor-data';
+import Users from '/imports/api/users';
+import Auth from '/imports/ui/services/auth/index';
+import JoinVideoOptions from './component';
+
+const JoinVideoOptionsContainer = props => (<JoinVideoOptions {...props} />);
+
+export default createContainer((params) => {
+  const userId = Auth.userID;
+  const user = Users.findOne({ userId: userId });
```

Comment:
Could you add a `service.js` file next to this one and move this call there, please? Together with the Auth and Users imports.

`service.js`:

```
import Users from '/imports/api/users';
import Auth from '/imports/ui/services/auth/index';

const isSharingVideo() {
  const userId = Auth.userID;
  const user = Users.findOne({ userId: userId });
  return user.has_stream ? true : false;
};

export default {
  isSharingVideo
};
```

And here:

```
import VideoMenuService from './service';
...
export default createContainer((params) => {
  const isSharingVideo = VideoMenuService.isSharingVideo();

  return {
  ...
```